docs/automations/assign-code-experts/README.md

```diff
@@ -6,10 +6,20 @@ When someone applies a `suggest-reviewers` label to a PR, use codeExperts to ass
 ![Assign Code Experts](assign_code_experts.png)
 
```

Comment:
This is the wrong screesnhot of gitBlame, should switch with the 🥷 codeExperts

Reply:
I put together a new image for this in the follow up commit.